diff --git a/logs/readme b/logs/readme
deleted file mode 100644
index 8b13789..0000000
--- a/logs/readme
+++ /dev/null
@@ -1 +0,0 @@
-
diff --git a/mmloc_keras.py b/mmloc_keras.py
index 1dcf3b9..f2b56bf 100644
--- a/mmloc_keras.py
+++ b/mmloc_keras.py
@@ -24,9 +24,12 @@ output_dim = 2
 num_epochs = 100
 learning_rate = 0.001
 
+model_name = "mmloc_bucharest"
 
-wandb.init(entity="mmloc",project="mmloc_edinburgh",sync_tensorboard=True,
-           config={"epochs": num_epochs,"batch_size": batch_size,    
+wandb.init(entity="mmloc",project=model_name,sync_tensorboard=True,
+           config={"epochs": num_epochs,"batch_size": batch_size,"hidden_size":hidden_size,
+                   "learning_rate":learning_rate,"sensor_input_size":sensor_input_size,
+                   "wifi_input_size":wifi_input_size,"output_dim":output_dim
                    }
            )
 
@@ -66,7 +69,7 @@ mmloc=Model(inputs=[sensorinput,wifiinput],outputs=[output])
 mmloc.compile(optimizer=RMSprop(learning_rate),
                  loss='mse',metrics=["acc"])
 
-model_name = "mmloc_multi_model"
+
 tensorboard = TensorBoard(log_dir='logs/{}'.format(model_name))
 
 mmloc.fit([SensorTrain,WifiTrain], locationtrain,
@@ -76,8 +79,8 @@ mmloc.fit([SensorTrain,WifiTrain], locationtrain,
                        )
 
 #save model
-mmloc.save("romaniamodel/mmloc_multi.h5")
-mmloc.save(os.path.join(wandb.run.dir, "wanbd_mmloc_multi.h5"))
+mmloc.save("romaniamodel/"+str(model_name)+".h5")
+mmloc.save(os.path.join(wandb.run.dir, "wanbd_"+str(model_name)+".h5"))
 fig=plt.figure()
 locPrediction = mmloc.predict([SensorTest,WifiTest], batch_size=100)
 aveLocPrediction = pf.get_ave_prediction(locPrediction, 100)
@@ -86,6 +89,6 @@ plt.plot(data[:,0],data[:,1],'b',data[:,2],data[:,3],'r')
 plt.legend(['target','prediction'],loc='upper right')
 plt.xlabel("x-latitude")
 plt.ylabel("y-longitude")
-plt.title('mmloc_multi prediction')
-fig.savefig("romaniapredictionpng/mmloc_multi_locprediction.png")
-wandb.log({"chart": wandb.Image("romaniapredictionpng/mmloc_multi_locprediction.png")})
\ No newline at end of file
+plt.title(str(model_name)+" Prediction")
+fig.savefig("romaniapredictionpng/"+str(model_name)+"_locprediction.png")
+wandb.log({"chart": wandb.Image("romaniapredictionpng/"+str(model_name)+"_locprediction.png")})
\ No newline at end of file
diff --git a/model/mmloc_multi.h5 b/model/mmloc_multi.h5
deleted file mode 100644
index 4eff092..0000000
Binary files a/model/mmloc_multi.h5 and /dev/null differ
diff --git a/model/mmloc_multi_residual.h5 b/model/mmloc_multi_residual.h5
deleted file mode 100644
index 63fae2e..0000000
Binary files a/model/mmloc_multi_residual.h5 and /dev/null differ
diff --git a/model/readme b/model/readme
deleted file mode 100644
index 8b13789..0000000
--- a/model/readme
+++ /dev/null
@@ -1 +0,0 @@
-
diff --git a/model/sensor_baseline_model.h5 b/model/sensor_baseline_model.h5
deleted file mode 100644
index 8dcf5b8..0000000
Binary files a/model/sensor_baseline_model.h5 and /dev/null differ
diff --git a/model/sensor_downsample_model.h5 b/model/sensor_downsample_model.h5
deleted file mode 100644
index 90ab343..0000000
Binary files a/model/sensor_downsample_model.h5 and /dev/null differ
diff --git a/model/wifi_DNN_model.h5 b/model/wifi_DNN_model.h5
deleted file mode 100644
index 967816e..0000000
Binary files a/model/wifi_DNN_model.h5 and /dev/null differ
diff --git a/model/wifi_DNN_nonzero_model.h5 b/model/wifi_DNN_nonzero_model.h5
deleted file mode 100644
index 0c52772..0000000
Binary files a/model/wifi_DNN_nonzero_model.h5 and /dev/null differ
